api/VERSION

```diff
@@ -1 +1 @@
-v5
+v6
```

Comment:
Add a README under api/ about the instructions of adding api-breaking change?

Reply:
Will do this as a separate pr with the script to help version change.

Reply:
+1. Some questions that should be clarified:

- When we move from v6 to v7, do we keep the v6 folder and make a new v7 folder? Or completely move v6 to v7?
- Will there be some script to do the replacements?